.pullapprove.yml

```diff
@@ -0,0 +1,29 @@
+version: 3
+
+pullapprove_conditions:
+- "'WIP' not in title"
+- "'ci/circleci: build' in statuses.succeeded"
+
+notifications:
+- when: pull_request.opened
+  if: "author_association in ['FIRST_TIMER', 'FIRST_TIME_CONTRIBUTOR', 'CONTRIBUTOR']"
+  comment: >
+    Hey @{{ author }}, please make sure your tests pass and you have looked everything over. Then we will review.
+
+    Thanks for the PR!
+- when: pullapprove.approved
+  comment: "The review is completed. @{{ author }}, please merge this in when ready."
+
+groups:
+  code:
+    reviewers:
+      users:
+      - lomudi
```

Comment:
If the team has a group named `applicaster/dev-rel` there is no need to set those users.

Reply:
done